### DISCUSSION TITLE
Disable Metadata LUTS on all BRAW Toolbox Clips in Timeline?

### DISCUSSION THREAD

**Discussion by tangierc** [latenitefilms/BRAWToolbox#135] [General]
Hello BRAW Toolbox community. Is there a way to disable all LUTs for BRAW Toolbox clips in the timeline?  This isn't a global setting in the inspector and the only way I can see doing it right now is going opening each clip, selecting the generator, and disabling it in the BRAW Toolbox Show Parameter inspector under the Clip Metadata section. 

I need to disable the camera LUT for every BRAW Toolbox clip in my timeline.

**Comment by latenitefilms** [comment]
AFAIK, there's no way currently to do this. One solution would be to export a FCPXML, do a find and replace, then import back in.

**Reply by tangierc** [reply]
I literally was just about to do that in BBEdit Chris. The one worry I have when working with XMLs is I never know everything that is/isn't maintained without watching everything again.  A gang feature in FCP would be amazing to compare before and after between the browser and timeline. 

Do you know a webpage that has a simple list of all of the things that FCP maintains in the XML. Oddly, that simple resource is always difficult to find with a google search. I haven't found it on the developer site. Intelligent Assistance's SendtoX manual has a nice listing for that product, but I don't need that.

Between generators, titles, adjustment layers (aka titles), speed changes, stabilization, masks, etc. I don't have it committed to memory what all is for sure going to come back in if I do it this way and what I'll  have to regenerate. It's a confidence issue ;)

Maybe this resource would be good on fcp.cafe

**Reply by latenitefilms** [reply]
It's mainly Titles and things like Effect/Generator Drop Zones that don't come across correctly in FCPXML.

Good idea about adding this info to FCP Cafe! Will do.

My advice would be to just have a timeline with ONLY the BRAW clips, fix that, then recombine with your original timeline.

The fix for your Synchronised Clips is also now on the App Store.

You can bring in a MOV export of your film and put it on top of everything with a Difference Blend Mode to easily visually spot any differences.

**Comment by tangierc** [comment]
Maybe a section on FCP.cafe for specific error codes too. Those can be difficult to find. I just got yet another export error... (error -536870181). This is one of several I’ve gotten in recent weeks. It seems I’ve had more export errors in the past few months than I’ve had in years of using FCP (X) so a repository of what these codes are / mean could be useful.


Tangier

**Reply by latenitefilms** [reply]
**Error -536870181** could be anything. It's not a specific error code - you'd need a lot more context to work out what it means.
